rest/route/github.go

```diff
@@ -721,19 +721,18 @@ func (gh *githubHookApi) AddIntentForPR(ctx context.Context, pr *github.PullRequ
 	if projectRef == nil {
 		return nil
 	}
-	var mergeBase string
-	if projectRef.OldestAllowedMergeBase != "" {
-		ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
-		defer cancel()
-		baseOwnerAndRepo := strings.Split(pr.Base.Repo.GetFullName(), "/")
-		if len(baseOwnerAndRepo) != 2 {
-			return errors.New("PR base repo name is invalid (expected [owner]/[repo])")
-		}
-		mergeBase, err = thirdparty.GetGithubMergeBaseRevision(ctx, baseOwnerAndRepo[0], baseOwnerAndRepo[1], pr.Base.GetRef(), pr.Head.GetRef())
-		if err != nil {
-			return errors.Wrapf(err, "getting merge base between branches '%s' and '%s'", pr.Base.GetRef(), pr.Head.GetRef())
-		}
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
```

Comment:
Should we still use this separate context, rather than just use the passed-in one? I imagine it was probably here to deal with the fact that GitHub occasionally will just randomly cancel a webhook request, but that was worked around by [adding a separate context to the webhook route](https://github.com/evergreen-ci/evergreen/blob/8d83306130dd81d6bf8bcc92b8edf155b4f9d842/rest/route/github.go#L133-L137).

Reply:
Ah yep, will remove. Thanks